.editorconfig

```diff
@@ -18,6 +18,129 @@ indent_style = space
 indent_size = 4
 indent_style = space
 
-[*.yaml]
-indent_size = 2
+[*.cs]
+indent_size = 4
 indent_style = space
+
+# New line preferences
+csharp_new_line_before_open_brace = all
```

Comment:
Is there a place where i can see an example of these styles?

Reply:
Yes, here is [a link](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/csharp-formatting-options).